src/word-search/index.js

```diff
@@ -1,3 +1,47 @@
-function wordSearch(words, word) {}
+function wordSearch(words, word) {
+
+  word = word.split('');
+
+  for (let i=0; i<words.length; i++){
+    for (let j=0; j<words[i].length; j++){
+      if(words[i][j] === word[0]){
+        checkWord(words, word, i, j, k=0);
```

Comment:
Where did k come from?

Reply:
it represents 'count' in the checkWord function.